core/theme/ThemeManager.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+namespace luya\theme;
+
+use luya\base\PackageConfig;
+use luya\Exception;
+use luya\helpers\Json;
+use Yii;
+use yii\base\InvalidArgumentException;
+use yii\base\InvalidConfigException;
+
+/**
+ * Core theme manager for LUYA.
+ *
+ * This component manage available themes via file system and the actual display themes.
+ *
+ * @author Mateusz Szymański Teamwant <zzixxus@gmail.com>
+ * @author Bennet Klarhölter <boehsermoe@me.com>
+ * @since  1.1.0
+ */
+class ThemeManager extends \yii\base\Component
+{
+    const APP_THEMES_BLANK = '@app/themes/blank';
```

Comment:
What is the idea behind APP_THEMES_BLANK? It acts like a default theme? If i don't define a theme, why do i need a default theme? I don't understand this one.

Reply:
The APP_THEMES_BLANK was used as a default/fallback theme. But now the theme setup will check of a activated theme, so this is no longer needed.